internal/pkg/config/flags_shared.go

```diff
@@ -76,3 +78,26 @@ func GetClientIDsParamReq(desc string) (paramReq ParameterRequirement) {
 		IsRequired: true,
 	}
 }
+
```

Comment:
Line 50: `Description: "Never prompt when missing parameters",` 
This seems to apply to yes/no confirmations also.
Consider a description like "Never prompt when missing parameters, answer all y/n questions with y"

Reply:
ok. done.